spyderlib/widgets/calltip.py

```diff
@@ -211,7 +211,7 @@ def show_tip(self, point, tip, wrapped_tiplines):
                     vertical = 'top'
             else:
                 vertical = 'top'
-        if point.x() + tip_width > screen_rect.width():
+        if point.x() + tip_width > screen_rect.width() + screen_rect:
```

Comment:
I say it because I'm seeing this traceback in your branch:

```python-traceback
  File "/home/carlos/Projects/spyder/github-repo/spyderlib/widgets/calltip.py", line 214, in show_tip
    if point.x() + tip_width > screen_rect.width() + screen_rect:
TypeError: unsupported operand type(s) for +: 'int' and 'QRect'
```